tests/contentmethods/test_methods.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import os
+import json
+import pytest
+from contentmethods import methods
+
+data_dir_path = os.path.join(os.path.dirname(__file__), "data")
+
+#input test data
+input_test_data = json.load(open(os.path.join(data_dir_path, "input.json")))
```

Comment:
Why is this not in the test function?

Reply:
Because data would be stored in same file for current test method and all upcoming test methods in this test_methods.py. So instead of reading again and again for every test method why not read it once and use it in all methods. This is my logic any change suggestion please let me know.

Reply:
As you said i have removed data folder and instead added i/p and o/p data inside test functions only.